ops/prod/api.tf

```diff
@@ -3,7 +3,7 @@ module "simple_report_api" {
   name   = "${local.name}-api"
   env    = local.env
 
-  instance_count = 3
+  instance_count = 4
```

Comment:
Can we make this a separate commit? It's unrelated to adding `terraform plan`.

Reply:
Absolutely! Done. I'll put that right-size in a separate PR.